boxes/boxes/react/src/pages/contract.tsx

```diff
@@ -1,8 +1,8 @@
 import { useState } from 'react';
-import { Contract } from '@aztec/aztec.js';
+import { Contract, FunctionType } from '@aztec/aztec.js';
 import { useNumber } from '../hooks/useNumber';
 
-const IGNORE_FUNCTIONS = ['constructor', 'process_log', 'sync_notes'];
```

Comment:
The `process_log` entrypoint function on contracts no longer exists. Forgot to remove this in my other PR where I was streamlining log processing.